source/production/F0.Minesweeper.Components/Cell.razor.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Threading.Tasks;
+using F0.Minesweeper.Components.Abstractions;
+using Microsoft.AspNetCore.Components;
+
+namespace F0.Minesweeper.Components
+{
+	public partial class Cell
+	{
+		[Parameter]
+		public Location Location { get; set; }
+
+		protected override void OnParametersSet()
+		{
+			if (Location == null) 
+			{
+				throw new ArgumentNullException(nameof(Location));
+			}
+		}
+
+		private Task OnClickAsync()
```

Comment:
Can we maybe change the issues requirement so that e.g. on click the cells text changes from covered to uncovered? because with the console output I couldn't really add tests for the click event.

Reply:
Sure, sounds better to me too.

Reply:
It's implemented!